src/libraries/System.Net.HttpListener/src/System/Net/Managed/HttpConnection.cs

```diff
@@ -43,33 +44,33 @@ internal sealed class HttpConnection
     {
         private static AsyncCallback s_onreadCallback = new AsyncCallback(OnRead);
         private const int BufferSize = 8192;
-        private Socket _socket;
+        private Socket? _socket;
         private Stream _stream;
         private HttpEndPointListener _epl;
-        private MemoryStream _memoryStream;
-        private byte[] _buffer;
+        private MemoryStream? _memoryStream;
```

Comment:
is it necessary to mark `_memoryStream` as nullable? It is initialised in the `Init()` method

Reply:
Correct, but later in the file (line 217), `_memoryStream` is being set to `null`.